src/queueserver.ts

```diff
@@ -450,3 +450,24 @@ export const modifyQueue = async(op: QueueOps): Promise<IPlanModify> => {
     console.log(res);
     return res.data;
 }
+
+export const incrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
+    const res = await axiosInstance.post(`/queue/item/move`,
+        {
+            uid: item1_uid,
+            after_uid: item2_uid
+
+        });
+    console.log(res);
+    return res.data;
+}
+
+export const decrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
```

Comment:
Same question about the naming here.

Reply:
Updated

Reply:
Thank you for making it more consistent.